lib/vagrant-service-manager/binary_handlers/cdk_openshift_binary_handler.rb

```diff
@@ -1,7 +1,13 @@
 module VagrantPlugins
   module ServiceManager
-    class CDKOpenshiftBinaryHandler < CDKBinaryHandler
+    # Currently client binary installation of openshift for CDK is same as ADB
+    class CDKOpenshiftBinaryHandler < ADBOpenshiftBinaryHandler
+      # Default to latest stable origin oc version for CDK as it is different than
+      # OSE oc version running inside CDK development environment
+      LATEST_OC_VERSION = '1.2.1'.freeze
```

Comment:
> Does that mean that whenever a new OSE comes along we need to new service-manager release as well?

@hferentschik  No as @LalatenduMohanty  said most of latest client from oc should work with latest OSE. 
Now, to update this version, we can decide after how many iterations we should update.

Anyway, user always has a way to install the latest binary with `--cli-version`.